services/route_manager/src/main/java/com/futurewei/alcor/route/exception/ExistMultipleVpcRouter.java

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.route.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code= HttpStatus.INTERNAL_SERVER_ERROR, reason="exist multiple vpc router searched by vpc id")
+public class ExistMultipleVpcRouter extends Exception {
```

Comment:
Please update class name and description based on the comment above.

Reply:
NP